Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -130,6 +130,7 @@ - (void)finalizeInit
     self.rageShakeManager = [RageShakeManager sharedManager];
     self.encryptionTrustLevel = UserEncryptionTrustLevelUnknown;
     
+    securityActionsArray = [[NSMutableArray alloc] init];
```

Comment:
Could `securityActionsArray` be immutable?
His content is static. At least, the code manages it statically.